src/home.py

```
@@ -0,0 +1,296 @@
+        """
+            <div style="text-align: center; margin-top:300px">
+                <h2>Chat with AI</h2>
+            </div>
+            """,
+        unsafe_allow_html=True,
+    )
+
+    ai_user_message = st.text_area("Type your message to AI:", value=st.session_state.ai_user_message, height=100, key="ai_message")
+
+    if st.button("Send", key="send_ai_button"): #Eindeutiger Key
+        if ai_user_message.strip():
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)